pb/proof.proto

```diff
@@ -3,9 +3,18 @@ syntax="proto3";
 package proof.pb;
 
 message Proof {
+  // Start index of the leaves that match the queried namespace.ID.
   int64 start     = 1;
+  // End index (non-inclusive) of the leaves that match the queried
+  // namespace.ID.
   int64 end       = 2;
+  // Nodes hold the tree nodes necessary for the Merkle range proof.
   repeated bytes nodes = 3;
+ // LeafHash contains the namespace.ID if NMT does not have it and
+  // it should be proven. leafHash is necessary to prove the Absence Proof.
+  // This field will be empty in case of Inclusion Proof.
```

Comment:
[nit]

```suggestion
  // LeafHash contains the namespace.ID if NMT does not have it and
  // it should be proven. LeafHash is necessary to prove the Absence Proof.
  // This field will be empty in case of Inclusion Proof.
```